docs/sovereign/setup.md

```diff
@@ -45,100 +43,94 @@ Before proceeding, ensure that a SSH key for GitHub is configured on your machin
 :::note
 The prerequisites script verifies and downloads the necessary packages to run the nodes and clones the required repositories:
 
-- mx-chain-deploy-go: Initializes the configuration for the chain and deployment parameters.
-- mx-chain-proxy-go: Repository for the proxy.
-- mx-chain-sovereign-bridge-go: Repository for the bridge service.
+- **mx-chain-deploy-go**: Initializes the configuration for the chain and deployment parameters.
+- **mx-chain-proxy-go**: Repository for the proxy.
+- **mx-chain-sovereign-bridge-go**: Repository for the cross-chain service.
 :::
 
-## Step 4: Deploy Contracts on Testnet
+## Step 3: Deploy Sovereign setup
 
 First navigate to the sovereignBridge folder:
 
 ```bash
-cd ./sovereignBridge
+cd sovereignBridge
 ```
 
-1. Update the `/config/configs.cfg` file with the Owner and WASM file details. Example:
+1. Update the configuration file `config/configs.cfg` with paths you want to use, wallet location and main chain constants. Example:
     ```ini
+    # Sovereign Paths
+    SOVEREIGN_DIRECTORY="~/sovereign"
+    TXS_OUTFILE_DIRECTORY="${SOVEREIGN_DIRECTORY}/txsOutput"
+    CONTRACTS_DIRECTORY="${SOVEREIGN_DIRECTORY}/contracts"
```

Comment:
Do you have to save your contracts here? Or do the prerequisites do it automatically for you?